src/panels/config/zone/ha-config-zone.ts

```diff
@@ -515,40 +528,24 @@ export class HaConfigZone extends SubscribeMixin(LitElement) {
         flex-grow: 1;
         height: 100%;
       }
-      .flex paper-listbox,
+      .flex mwc-list,
       .flex .empty {
         border-left: 1px solid var(--divider-color);
         width: 250px;
         min-height: 100%;
         box-sizing: border-box;
       }
-      paper-icon-item {
-        padding-top: 4px;
-        padding-bottom: 4px;
-        cursor: pointer;
-      }
-      .overflow paper-icon-item:last-child {
-        margin-bottom: 80px;
-      }
-      paper-icon-item.iron-selected:before {
-        position: absolute;
-        top: 0;
-        right: 0;
-        bottom: 0;
-        left: 0;
-        pointer-events: none;
-        content: "";
-        background-color: var(--sidebar-selected-icon-color);
-        opacity: 0.12;
-        transition: opacity 15ms linear;
-        will-change: opacity;
-      }
       ha-card {
         margin-bottom: 100px;
       }
       ha-card paper-item {
         cursor: pointer;
       }
+      div[slot="meta"] {
```

Comment:
Is this needed 🤔 

I think you should set `mdc-list-item-meta-size` instead:

```
--mdc-list-item-meta-size: 48px;
```

Reply:
It's needed, since we don't use buttons in a mwc-list a lot. The button gets pushed to below without: 
![image](https://github.com/home-assistant/frontend/assets/32477463/6559df69-d13c-4fbe-ad3e-601073fed615)

Setting the meta-size also works. Replaced.